services/centralized-kubecost/0.34.0/defaults/cm.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
 data:
   values.yaml: |
     ---
+    priorityClassName: "dkp-high-priority"
```

Comment:
(applies to the below kubecost values as well) we use an umbrella chart that brings in upstream kubecost ("cost-analyzer" helm chart) as a subchart -- so all the upstream values are nested under `cost-analyzer` (line 10 in this diff here).

The upstream helm chart values are here: https://github.com/kubecost/cost-analyzer-helm-chart/blob/develop/cost-analyzer/values.yaml there are currently 2 places where priorityClassName can be set, but I am seeing now that these are both components that are not enabled by default. I'm going to actually look at contributing a PR upstream to add support to set priority class on all the deployments. Let's hold off on the 2 kubecost apps here until I get that in!

Reply:
https://github.com/kubecost/cost-analyzer-helm-chart/blob/1723242ea6744b2d20cdda6dea9a4a59ffe22faa/cost-analyzer/values.yaml#L487-L490

Took a closer look at kubecost as I went in to make the changes -- they do actually support creating a priority class and using it in some (but not all) deployments. We can provide our own priority class name and it'll get used:

```
priority:
  enabled: true
  name: dkp-high-priority
```

I will still look into seeing whether I should open an upstream pr to allow setting it for all deployments not just the main kubecost ones but we can set this field now without waiting! This should be set in both kubecost and centralized-kubecost

Reply:
Good catch! Updates have been made